nixos/modules/system/boot/luksroot.nix

```diff
@@ -316,6 +317,19 @@ in
             '';
           };
 
+          keyFileOffset = mkOption {
+            default = null;
+            example = 4096;
+            type = types.nullOr types.int;
+            description = ''
+              The offset of the key file. Use this  in combination with
```

Comment:
Minor nitpicks, but:
- You've got an extra space after "Use this"
- "use part of a file as key" - do you mean "as the key"?
- "is used as key file" - you may mean "used as a key file"?

Reply:
I fixed what I think was wrong. "as the key" would be wrong because it's not actually used as key, but as key file. I hope you are happy with the changed version